source/configure/database-configuration-settings.rst

```diff
@@ -388,27 +388,111 @@ Replica lag settings
+
+          .. code-block:: json
+            :emphasize-lines: 4,5,6,7,8
+    
+            {
+              "SqlSettings": {
+                  "ReplicaLagSettings": [
+                    {
+                        "DataSource": "replica-1",
+                        "QueryAbsoluteLag": "select member_id, count_transactions_remote_in_applier_queue FROM performance_schema.replication_group_member_stats where member_id=<>",
+                        "QueryTimeLag": ""
+                    }
+                  ]
+              }
+            }
+
+    .. tab:: PostgreSQL replication slots
+
+      1. Add the configuration highlighted below to your ``SqlSettings.ReplicaLagSettings`` array. This query should run against the **primary** node in your cluster, to do this change the ``DataSource`` to match the `SqlSettings.DataSource <#data-source>`__ setting you have configured. 
+
+          For more information on pg_stat_replication, see the `PostgreSQL documentation <https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-PG-STAT-REPLICATION-VIEW>`__.
+
+              **Example:**
+
+              .. code-block:: json
+                :emphasize-lines: 4,5,6,7,8
+        
+                {
+                  "SqlSettings": {
+                      "ReplicaLagSettings": [
+                        {
+                            "DataSource": "postgres://mmuser:password@localhost:5432/mattermost_test?sslmode=disable&connect_timeout=10.",
+                            "QueryAbsoluteLag": "select usename, pg_wal_lsn_diff(pg_current_wal_lsn(),replay_lsn) as metric from pg_stat_replication;",
+                            "QueryTimeLag": ""
+                        }
+                      ]
+                  }
+                }
+
+      2. Grant permissions to the database user for ``pg_monitor``. This user should be the same user configured above in the ``DataSource`` string.
+
+          For more information on roles, see the `PostgreSQL documentation <https://www.postgresql.org/docs/10/default-roles.html>`__.
+
+          .. code-block:: bash
+
+            sudo -u postgres psql
+            postgres=# GRANT pg_monitor TO mmuser;
+
+      
+2. Save the config and restart all Mattermost nodes.
+3. Navigate to your Grafana instance monitoring Mattermost and open the `Mattermost Performance Monitoring v2 <https://grafana.com/grafana/dashboards/15582-mattermost-performance-monitoring-v2/>`__ dashboard.
+4. The ``QueryTimeLag`` chart is already setup for you utilizing the existing ``Replica Lag`` chart. If using ``QueryAbsoluteLag`` metric clone the ``Replica Lag`` chart and edit the query to use the below absolute lag metrics and modify the title to be ``Replica Lag Absolute``.
+
+    .. code-block:: none
+
+      mattermost_db_replica_lag_abs{instance=~"$server"}
+
+    .. image::  ../../source/images/database-configuration-settings-replica-lag-grafana-1.jpg
+      :alt: A screenshot showing how to clone a chart within Grafana
+
```

Comment:
I don't know how you'd suggest adding spaces here between the images. The images may not be needed, but I felt it added some clarity.